addons/mail/static/tests/systray/systray_activity_menu_tests.js

```diff
@@ -125,5 +142,48 @@ QUnit.test('activity menu widget: activity menu with 3 records', function (asser
 
     activityMenu.destroy();
 });
+
+QUnit.test('activity menu widget: activity view icon', function (assert) {
+    /* Check that the widget will correctly create the link */
+    assert.expect(7);
+    var self = this;
+    var activityMenu = new ActivityMenu();
+    testUtils.addMockEnvironment(activityMenu, {
+        services: this.services,
+        mockRPC: function (route, args) {
+            if (args.method === 'systray_get_activities') {
+                return $.when(self.data['mail.activity.menu']['records']);
+            }
+            return this._super(route, args);
+        },
+    });
+    activityMenu.appendTo($('#qunit-fixture'));
+    assert.strictEqual(activityMenu.$('.o_mail_activity_action').length, 2, "widget should have 2 activity view icons");
+
+    var first = activityMenu.$('.o_mail_activity_action').eq(0);
+    var second = activityMenu.$('.o_mail_activity_action').eq(1);
+    assert.strictEqual(first[0].attributes['data-model_name'].value, "Issue", "first activity action should link to 'Issue'");
+    assert.strictEqual(first.hasClass('fa-th'), true, "wrong activity action icon");
```

Comment:
- Please formulate assertion comments as _"should ..."_ instead of _"wrong ..."_, Here, something like _"should display the activity action icon"_.
- Use `fa-clock-o`
- `assert.ok(..., ...)` is equivalent to `assert.strictEqual(..., true, ...)`